iroh/src/endpoint.rs

```diff
@@ -952,6 +952,22 @@ impl Endpoint {
 
     // # Methods for terminating the endpoint.
 
+    /// Closes the QUIC endpoint and the magic socket.
+    ///
+    /// This will close all open QUIC connections.
+    ///
+    /// It will then wait for all connections to actually be shutdown, and afterwards close
+    /// the magic socket.  Be aware however that the underlying UDP sockets are only closed
+    /// on [`Drop`], bearing in mind the [`Endpoint`] is only dropped once all the clones
+    /// are dropped.
+    ///
+    /// Returns an error if closing the magic socket failed.
+    /// TODO: Document error cases.
+    pub async fn close(&self) -> Result<()> {
+        self.close_with_code(1u16.into(), b"shutting down").await?;
```

Comment:
suggestions welcome here for code and and message

Reply:
I would suggest `0` and `b""`.  That is the same behaviour you get when dropping the endpoint or dropping a connection.